src/Compilers/CSharp/Test/Semantic/Semantics/Utf8StringsLiteralsTests.cs

```diff
@@ -1786,5 +2023,639 @@ static void Main()
+using System;
+class C
+{
+    static void Main()
+    {
+        System.Console.WriteLine(Test(""s""u8));
+    }
+
+    static string Test(ReadOnlySpan<byte> a) => ""ReadOnlySpan"";
+    static string Test(byte[] a) => ""array"";
+}
+";
+            var comp = CreateCompilation(source, targetFramework: TargetFramework.NetCoreApp, options: TestOptions.DebugExe);
+            CompileAndVerify(comp, expectedOutput: @"array").VerifyDiagnostics();
+        }
+
+        [ConditionalFact(typeof(CoreClrOnly))]
+        public void OverloadResolution_12()
+        {
+            var source = @"
+using System;
+class C
+{
+    static void Main()
+    {
+        System.Console.WriteLine(Test(""s""u8));
+    }
+
+    static string Test(Span<byte> a) => ""Span"";
+    static string Test(byte[] a) => ""array"";
+}
+";
+            var comp = CreateCompilation(source, targetFramework: TargetFramework.NetCoreApp, options: TestOptions.DebugExe);
+            CompileAndVerify(comp, expectedOutput: @"array").VerifyDiagnostics();
+        }
+
+        [ConditionalFact(typeof(CoreClrOnly))]
+        public void OverloadResolution_13()
+        {
+            var source = @"
+using System;
+class C
+{
+    static void Main()
+    {
+        System.Console.WriteLine(Test(""s""u8));
+    }
+
+    static string Test(ReadOnlySpan<byte> a) => ""ReadOnlySpan"";
+    static string Test(Span<byte> a) => ""Span"";
+}
+";
+            var comp = CreateCompilation(source, targetFramework: TargetFramework.NetCoreApp, options: TestOptions.DebugExe);
+            CompileAndVerify(comp, expectedOutput: @"Span").VerifyDiagnostics();
+        }
+
+        [ConditionalFact(typeof(CoreClrOnly))]
+        public void OverloadResolution_14()
+        {
+            var source = @"
+using System;
+class C
+{
+    static void Main()
+    {
+        System.Console.WriteLine(Test(""s""u8));
+    }
+
+    static string Test(ReadOnlySpan<byte> a) => ""ReadOnlySpan"";
+    static string Test(byte[] a) => ""array"";
+    static string Test(Span<byte> a) => ""Span"";
+}
+";
+            var comp = CreateCompilation(source, targetFramework: TargetFramework.NetCoreApp, options: TestOptions.DebugExe);
+            CompileAndVerify(comp, expectedOutput: @"array").VerifyDiagnostics();
+        }
+
+        [ConditionalFact(typeof(CoreClrOnly))]
+        public void UserDefinedImplicitConversions_03()
+        {
+            var source = @"
+using System;
+class C
+{
+    static void Main()
+    {
+        C1 x = ""hello""u8;
+    }
+}
+
+class C1
+{
+    public static implicit operator C1(byte[] x)
+    {
+        Helpers.Print(x);
+        return new C1();
+    }
+}
+";
+            var comp = CreateCompilation(source + HelpersSource, targetFramework: TargetFramework.NetCoreApp, options: TestOptions.DebugExe);
+
+            CompileAndVerify(comp, expectedOutput: @"
+{ 0x68 0x65 0x6C 0x6C 0x6F }
+").VerifyDiagnostics();
+        }
+
+        [ConditionalFact(typeof(CoreClrOnly))]
```

Comment:
>[ConditionalFact(typeof(CoreClrOnly))] [](http://example.com/codeflow?start=8&length=38)

`[Fact]`

Reply:
HelpersSource refers to types that are not available on desktop. I am fairly comfortable to test this scenario only on CoreClr.